src/lib/stitching/exchange/v2/stitching.ts

```diff
@@ -166,6 +180,7 @@ export const exchangeStitchingEnvironment = ({
       buyerDetails: OrderParty
       sellerDetails: OrderParty
       creditCard: CreditCard
+      isInquiryOrder: Boolean!
```

Comment:
question:
Did you intentionally add `isInquiryOrder` under `CommerceBuyOrder` and `CommerceOrder` or it was needed to make stitching work?
I am not strongly against it though, maybe we start accepting `BUY` inquiry orders soon.

Reply:
Yes, it's required because it shared the CommerceOrder interface.